easystacks/software.eessi.io/2023.06/icelake_cclake/049-eb-4.9.4.yml

```
@@ -0,0 +1,5 @@
+# 049-eb-4.9.1.yml: total build duration = 512 minutes
+easyconfigs:
+  - /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/R/4.2.2-foss-2022b/easybuild/R-4.2.2-foss-2022b.eb:
+      options:
+        include-easyblocks: /cvmfs/software.eessi.io/versions/2023.06/software/linux/x86_64/intel/haswell/software/R/4.2.2-foss-2022b/easybuild/reprod/easyblocks/*.py
```

We're doing rebuilds of this R version with EB 4.9.4 (in #1062), so we can use the easyconfig+easyblock from that version here too.
```suggestion
  - R-4.2.2-foss-2022b.eb
```